src/systems/ec/implementations/vanilla/packages/ec-component-tag/ec-component-tag.scss

```
@@ -13,8 +13,9 @@
   .ecl-tag {
     align-items: center;
     background-color: map-get($ecl-colors, 'grey-15');
-    border-radius: (2 * $ecl-spacing-xs + $ecl-font-size-s * $ecl-line-height-s) /
-      2;
+    border-radius: (
+        2 * $ecl-spacing-xs + ($ecl-font-size-s * $ecl-line-height-s) / 1rem
```

```suggestion
        2 * $ecl-spacing-xs + $ecl-line-height-s
```

the line-height is now the right value (we don't need to take the font-size into account)